servant-client/src/Servant/Client/Internal/HttpClient.hs

```diff
@@ -73,11 +73,12 @@ data ClientEnv
   { manager :: Client.Manager
   , baseUrl :: BaseUrl
   , cookieJar :: Maybe (TVar Client.CookieJar)
+  , makeClientRequest :: BaseUrl -> Request -> Client.Request
```

Comment:
- this one needs haddocks
    - at leas mentioning that `cookieJar` is applied to the request created
- also we'll need to export default value from non-`Internal` module, as probably many implementations will call it either mangling servant's or http-client's `Request`. I propose `defaultMakeClientRequest` (it also needs haddocks, explaining what it does).

Reply:
Thanks for the comment I will do that tomorrow.